src/utils.js

```diff
@@ -1,13 +1,10 @@
-export const isMemberPropValid = members => {
-  if (members === undefined) {
-    return false
-  }
-
-  if (!(typeof members === 'string' || Array.isArray(members))) {
-    return false
-  }
+export const buildUrlWithParams = (url, params = {}) => {
+  const queryParams = Object.keys(params)
+    .filter((k) => params[k] !== undefined && params[k] !== null)
```

Comment:
I would change _k_ for some more meaningful name. Code readability helps maintainability for people that haven't touch the code..